libtransmission/session.cc

```diff
@@ -128,6 +128,54 @@ tr_peer_id_t tr_peerIdInit()
 ****
 ***/
 
+bool tr_session::LpdMediator::onPeerFound(std::string_view info_hash_str, tr_address address, tr_port port)
+{
+    if (auto digest = tr_sha1_from_string(info_hash_str); digest)
```

Comment:
nit: why not early return?

Reply:
Yeah, that is a little better. Updated in 75ef0e6b47f01d0d0c92499f018537a76ca1d470